typescript/aws-transfer-sftp-server/main.ts

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+import * as cdk from 'aws-cdk-lib';
+import { SftpServerStack, SftpServerStackProps } from './aws-transfer-sftp-server';
+
+// Follow the setup process at https://docs.aws.amazon.com/cdk/v2/guide/environments.html
+const props: cdk.StackProps = {
+  env: {
+    account: process.env.CDK_DEFAULT_ACCOUNT,
+    region: process.env.CDK_DEFAULT_REGION,
+  }
+};
+
+const userPublicKeys = [
```

Comment:
Nit: Something like this should be from Parameter Store or similar, and should be documented. For this example, hardcoding the SSH key is fine but customers should use context values, parameter store, secrets manager, etc. for holding things like this.